pkg/cmd/properties/properties.go

```diff
@@ -120,6 +121,7 @@ func (p *properties) AddStringSliceProperty(name string, defaultVal []string, de
 		flagName := p.nameToFlagName(name)
 		p.rootCmd.Flags().StringSlice(flagName, defaultVal, description)
 		p.bindOrPanic(name, p.rootCmd.Flags().Lookup(flagName))
+		p.rootCmd.Flags().MarkHidden(flagName)
```

Comment:
All properties are hidden?

Reply:
Almost. The help looks like this now:
Flags:
      --envFile string      Path of the file with environment variables to override configuration
  -h, --help                help for discovery_agent
      --pathConfig string   YAML configuration file path for the agent
      --status              Get the status of all the Health Checks
      --synchronize         Run the sync process for the discovery agent
  -v, --version             version for discovery_agent

envFile, pathConfig, status, and synchronize were either creating flags (not props) or were creating persistent properties (don't know what that diff is). So, the methods I changed are not affecting these.

Reply:
I toyed with passing a bool to hide/not hide to each of the methods, but that would bubble all over. Or, to copy/paste and have different methods completely to hide/not hide. I decided that for now my changes work.